services/kafka-operator/0.20.0/metadata.yaml

```diff
@@ -1 +1 @@
-k8sVersionSupport: ">=1.19"
+k8sVersionSupport: "1.19 - 1.24.x"
```

Comment:
@cbuto just to double check, is this true? just making sure it shouldn't be 1.19 - 1.19.x because of 0.20.2 which was added with >=1.20

Reply:
hmmm i'm not totally sure, i was looking at https://github.com/banzaicloud/koperator/releases/tag/v0.21.0 and there's a note that it added support for v1.23 🤔 

maybe this should be `1.19 - 1.22.x`?

Reply:
I saw that too.. but we also have 0.20.2 in the catalog which is <0.21.0 and that app is released in dkp 2.3/2.4 so I would think we had validated it against k8s 1.23/1.24? 😬

Reply:
oh okay the PR to add 1.23 support is just bumping dependencies, so 0.20.2 does work on 1.23